src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/FeatureChainingWfsTest.java

```diff
@@ -1487,14 +1487,13 @@ public void testNestedFilterEncodingDisabled() throws IOException, FilterToSQLEx
         AppSchemaDataAccessRegistry.getAppSchemaProperties().setProperty(
                 AppSchemaDataAccessConfigurator.PROPERTY_ENCODE_NESTED_FILTERS, "false");
         try {
+            assertFalse(AppSchemaDataAccessConfigurator.shouldEncodeNestedFilters());
+
             FeatureTypeInfo ftInfo = getCatalog().getFeatureTypeByName("gsml", "MappedFeature");
             FeatureSource fs = ftInfo.getFeatureSource(new NullProgressListener(), null);
             AppSchemaDataAccess da = (AppSchemaDataAccess) fs.getDataStore();
             FeatureTypeMapping rootMapping = da.getMappingByNameOrElement(ftInfo.getQualifiedName());
 
-            // make sure nested filters encoding is enabled, otherwise skip test
-            assumeTrue(shouldTestNestedFiltersEncoding(rootMapping));
-
             JDBCDataStore store = (JDBCDataStore) rootMapping.getSource().getDataStore();
```

Comment:
@bencaradocdavies yep, test should really be skipped when property files are used... I just pushed a commit that should fix it.